opm/simulators/timestepping/SimulatorTimer.hpp

```diff
@@ -121,6 +121,7 @@ namespace Opm
         std::vector<double> timesteps_;
         int current_step_;
         double current_time_;
+        double end_step_;
```

Comment:
Do you really want to have this as `double`?  Is it not supposed to be an integer, maybe even an `int`?